kraken-std/src/kraken/std/cargo/tasks/cargo_generate_deb.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+from dataclasses import dataclass
+from pathlib import Path
+
+from kraken.core import Property, Task, TaskStatus
+
+from kraken.std.descriptors.resource import LibraryArtifact
+
+
+@dataclass
+class CargoDebianArtifact(LibraryArtifact):
+    pass
+
+
+class CargoGenerateDebPackage(Task):
+    package_name: Property[str] = Property.default("CargoDebian")
```

Comment:
Why is this the default value? Don't we always want to force the caller to pass a package name (in which case there should not be a default)? 🤔

Reply:
That is a good shout!